app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -30,14 +29,22 @@ $large-input-size: 50px;
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
+
+  &:focus {
+    outline: $govuk-focus-width solid $govuk-focus-colour;
+    outline-offset: inherit;
```

Comment:
Probably out of scope, but the search box is long overdue an overhaul to make it look like GOVUK Frontend. We should talk about that.

Reply:
I agree about this needs an overhaul. Again, good call for an improvement.

Reply:
I think we should address this after V3 has been merged. Issue logged here: https://github.com/alphagov/govuk_publishing_components/issues/1027